NickvisionTubeConverter.GNOME/Controls/DownloadRow.cs

```diff
@@ -176,7 +158,11 @@ public void SetProgressState(DownloadProgressState state)
                 if (!_runPulsingBar)
                 {
                     _runPulsingBar = true;
-                    g_timeout_add(30, _pulsingBarCallback, 0);
+                    GLib.Functions.TimeoutAdd(0, 30, () =>
+                    {
+                        _pulsingBar.Pulse();
+                        return false;
```

Comment:
Shouldn't this be returning `_runPulsingBar`?

Reply:
Yes, let me fix that :)